### DISCUSSION TITLE
Version 2.0.9 hash has changed

### DISCUSSION THREAD

**Discussion by chriskapp** [composer/composer#9767] [General]
Hi, we are currently using composer 2.0.9 in our docker image and we check automatically the hash after download. We have noticed that the hash of the 2.0.9 release has changed from

`24faa5bc807e399f32e9a21a33fbb5b0686df9c8850efabe2c047c2ccfb9f9cc`
https://web.archive.org/web/20210203113506/https://getcomposer.org/download/

to

`8e91344a5ca2fc0fb583c50f195a1f36918908561c4ea3d6f01a4ef01c3b8560`
https://getcomposer.org/download/

is this an intentional change or could this be an security issue?

**Comment by stof** [comment]
see the discussion in https://github.com/composer/composer/issues/9746

**Reply by chriskapp** [reply]
ok, thanks for the info, then we will update the hashes accordingly
